index.js

```diff
@@ -0,0 +1,252 @@
+        "Backslash": ['\\','\|'],
+        'CapsLock':['CapsLk','CapsLk'],
+        'KeyA':['ф','Ф'],
+        'KeyS':['ы','Ы'],
+        'KeyD':['в','В'],
+        'KeyF':['а','А'],
+        'KeyG':['п','П'],
+        'KeyH':['р','Р'],
+        'KeyJ':['о','О'],
+        'KeyK':['л','Л'],
+        'KeyL':['д','Д'],
+        'Semicolon':['ж','Ж'],
+        'Quote': ['э','Э'],
+        'Enter': ['Enter','Enter'],
+        'ShiftLeft': ['Shift','Shift'],
+        'KeyZ': ['я','Я'],
+        'KeyX': ['ч','Ч'],
+        'KeyC': ['с','С'],
+        'KeyV': ['м','М'],
+        'KeyB': ['и','И'],
+        'KeyN': ['т','Т'],
+        'KeyM': ['ь','Ь'],
+        'Comma':['б','Б'],
+        'Period':['ю','Ю'],
+        'Slash':['\.','\,'],
+        'ShiftRight': ['Shift','Shift'],
+        "ControlLeft":['Ctrl','Ctrl'],
+        "AltLeft":['Alt','Alt'],
+        "Space":['Space','Space'],
+        "AltRight":['Alt','Alt'],
+        "ControlRight" :['Ctrl','Ctrl'] 
+    }
+} 
+let codeArray = [
+    ["Backquote", "Digit1", "Digit2", "Digit3", "Digit4", "Digit5", "Digit6", "Digit7", "Digit8", "Digit9", "Digit0", "Minus", "Equal", "Backspace"],
+    ["Tab", "KeyQ", "KeyW", "KeyE", "KeyR", "KeyT", "KeyY", "KeyU", "KeyI", "KeyO", "KeyP", "BracketLeft", "BracketRight", "Backslash"],
+    ["CapsLock", "KeyA", "KeyS", "KeyD", "KeyF", "KeyG", "KeyH", "KeyJ", "KeyK", "KeyL", "Semicolon", "Quote", "Enter"],
+    ["ShiftLeft", "KeyZ", "KeyX", "KeyC", "KeyV", "KeyB", "KeyN", "KeyM", "Comma", "Period", "Slash", "ShiftRight"],
+    ["ControlLeft", "AltLeft",'Space',"AltRight", "ControlRight"]
+];
+
+let body = document.getElementsByTagName('body')[0];
+body.appendChild(document.createElement('textarea'));
+let textarea = document.getElementsByTagName('textarea')[0];
+let section = document.createElement('section');
+section.className = 'virtual_keyboard';
+for (let i = 0; i < 5; i++){
+    let row = document.createElement('div');
+    row.className = `row ${i+1}`;
+    ////////ES6 features
+    section.appendChild(row);
+}
+body.appendChild(section);
+let capsShift = new Set();
+let lang;
+if (lang = localStorage.getItem('language')){
+
+}
+
+function initKeyboard(lang='lang1'){
+    localStorage.setItem('language',lang);
+    let i;
+    if (capsShift.has(16)||capsShift.has(20)){
+        i =1;
+    } else {
+        i=0;
+    }
+    for (let rowNum = 0; rowNum < 5; rowNum++){
+        let row = document.getElementsByClassName('row')[rowNum];
+        row.innerHTML = '';
+        for (let j=0; j<codeArray[rowNum].length; j++){
+            let key = document.createElement('div');
+            let keyCode = codeArray[rowNum][j];
+            key.className= `key`;
+            key.setAttribute('data',keyCode)
+            let keyValue = keyCodes[lang][keyCode];
+            key.innerHTML = keyValue[i];
+            row.appendChild(key);
+        }
+    }
+}
+
+initKeyboard(lang);
+
+let pressed = new Set();
+
+document.addEventListener('keydown',function(event){
```

Comment:
It's better to user named function instead anonymous. It's useful when we want to unsubscribe some handler.

```suggestion
document.addEventListener('keydown', handleKeyDown)
```